Dockerfile

```diff
@@ -168,10 +168,9 @@ RUN echo "source $PWD/hack/make/.integration-test-helpers" >> /etc/bash.bashrc
 
 # Get useful and necessary Hub images so we can "docker load" locally instead of pulling
 COPY contrib/download-frozen-image-v2.sh /go/src/github.com/docker/docker/contrib/
-# TODO: when issue #35963 fixed, we can upgrade the busybox to multi-arch
 RUN ./contrib/download-frozen-image-v2.sh /docker-frozen-images \
 	buildpack-deps:jessie@sha256:dd86dced7c9cd2a724e779730f0a53f93b7ef42228d4344b25ce9a42a1486251 \
-	busybox:latest@sha256:32f093055929dbc23dec4d03e09dfe971f5973a9ca5cf059cbfb644c206aa83f \
+	busybox:1.27-glibc@sha256:8c8f261a462eead45ab8e610d3e8f7a1e4fd1cd9bed5bc0a0c386784ab105d8e \
```

Comment:
@tianon are all these images now multi-arch? If so, we could unify these (perhaps even have a single Dockerfile for all arch's)

Reply:
Yes, absolutely -- I'd love to finally see a single `Dockerfile` for all architectures! :smile: :+1:

Reply:
Ah, I will +1 absolutely for this 👍  As the 1st step I can see is to merge the `Dockerfile.aarch64` into the `Dockerfile` as an experiment, then the firework is burning... 😄